content/faq/lbry-app-basics

```diff
@@ -0,0 +1,127 @@
+---
+title: LBRY App Basic User Guide
```

Comment:
I think this is more of a debugging FAQ than a basic user guide. Thoughts? Basic user guides to me would outline functionality and describe what is where, UI/UX, etc